ultralytics/utils/instance.py

```diff
@@ -385,6 +520,24 @@ def fliplr(self, w):
         if self.keypoints is not None:
             self.keypoints[..., 0] = w - self.keypoints[..., 0]
 
+    def fliplr_obb(self, w):
+        if self.obbData is None:
+            return
+        if len(self.obbData) == 0:
+            return
+        self.obbData[:, 0] = w-self.obbData[:, 1]
```

Comment:
❗ **CRITICAL**: Horizontal flip updates x-center using the y column: `self.obbData[:, 0] = w-self.obbData[:, 1]`. This looks like a copy/paste bug and will produce incorrect centers after fliplr.

**Suggested change:**
```suggestion
        self.obbData[:, 0] = w - self.obbData[:, 0]
```